frontend/src/utils.js

```diff
@@ -43,13 +43,56 @@ export const djikstra = (gameMap) => {
     throw new Error('No Path Found');
 }
 
+export const getSteps = (path, heightMap) => {
```

Comment:
We shouldn't have this be three.js aware either

Reply:
Not sure what you mean by this

Reply:
Like using x, y, z the intuitive way that you've mentioned above as opposed to what you'd use with three.js.

I think this means you've done the swap already:
```python
let [x,y,z] = [path[0][0], heightMap[path[0][0]][path[0][1]], path[0][1]];
```